src/Resources/contao/templates/nc_fieldset_duplication_html.html5

```diff
@@ -0,0 +1,20 @@
+<?php if ($this->values): ?>
+<table>
+    <thead>
+        <tr>
+            <?php foreach (array_keys($this->values[0]) as $name): ?>
+            <td><?= $this->labels[$name] ?? $name ?></td>
+            <?php endforeach ?>
+        </tr>
+    </thead>
+    <tbody>
+    <?php foreach ($this->values as $row): ?>
+        <tr>
+            <?php foreach ($row as $value): ?>
+            <td><?= $value ?></td>
+            <?php endforeach ?>
+        </tr>
+    <?php endforeach; ?>
+    </tbody>
+</table>
+<?php endif ?>
```

Comment:
I prefer the indentation that Contao uses for its template 😁 i.e. two spaces and:

```diff
 <?php foreach (array_keys($this->values[0]) as $name): ?>
-<td><?= $this->labels[$name] ?? $name ?></td>
+  <td><?= $this->labels[$name] ?? $name ?></td>
 <?php endforeach ?>
```

Reply:
Addressed with 64374a0a4078225513d890e28880ea99ca36b282